projects/control-service/projects/helm_charts/pipelines-control-service/templates/deployment.yaml

```diff
@@ -242,10 +244,13 @@ spec:
             - name: {{ $key }}
               value: "{{ $value }}"
             {{- end }}
+
           volumeMounts:
             - name: secrets
               mountPath: "/etc/secrets"
               readOnly: true
+            - mountPath: "/DATAJOBS_TEMP_STORAGE_FOLDER"
```

Comment:
is there a reason that the mounthPath is capitalized. I initially thought it's a env variable.

Reply:
no reason. I thought it made it a little more consistent everywhere. 
But I can change it.

Reply:
Will i change the lowercase? I have absolutely no problem doing that!

Reply:
If it's a consistency we are looking for - the name of the volumes and directories are in lowercase (e.g "datajob-template-volume") everywhere in the file. Only env. variable are upper case. 


I'd name the volume "datajobs-temp-storage-volume" 
 the folder "datajobs_temp_storage" (folder seems redundant in the name of a folder)
and the env variable "DATAJOBS_TEMP_STORAGE_FOLDER" (capitalized).